mlir/lib/Transforms/NormalizeMemRefs.cpp

```diff
@@ -517,6 +513,10 @@ Operation *NormalizeMemRefs::createOpResultsNormalized(FuncOp funcOp,
   // affine map, `oldOp` is returned without modification.
   if (resultTypeNormalized) {
     OpBuilder bb(oldOp);
+    for (auto &oldRegion : oldOp->getRegions()) {
+      auto newRegion = result.addRegion();
+      newRegion->getBlocks().splice(newRegion->begin(), oldRegion.getBlocks());
```

Comment:
Maybe 
```
Region* newRegion = result.addRegion();
newRegion->takeBody(oldRegion);
```

Note that the MLIR community would rather us spell out the `auto` unless the RHS already has the type (like it's a dyn_cast or something).

Reply:
Done.